server/src/db/migrations/20220222200523_rooms_table.ts

```diff
@@ -0,0 +1,39 @@
+import { Knex } from "knex";
+
+
+export async function up(knex: Knex): Promise<void> {
+    knex.schema.hasTable('Rooms').then(function(exists) {
+        if (!exists) {
+            return knex.schema.createTable('Rooms', function(t) {
+                t.increments('id').primary();
+                t.string("name", 100);
+            });
+        }
+    });
+
+}
+
+
+export async function down(knex: Knex): Promise<void> {
+    knex.schema.hasTable('Rooms').then(function(exists) {
+        if (!exists) {
+            return knex.schema.dropTable('Rooms');
+        }
+    });
+    knex.schema.hasTable('EquipmentForRooms').then(function(exists) {
```

Comment:
You can remove these now right?

Reply:
I thought I removed them, but I guess that also got reversed